spinalcordtoolbox/scripts/sct_get_centerline.py

```diff
@@ -178,6 +186,10 @@ def main(argv: Sequence[str]):
     else:
         printv("ERROR: The selected method is not available: {}. Please look at the help.".format(method), type='error')
         return
+    if arguments.centerline_soft == 1:
+        param_centerline.soft = 1
+    else:
+        param_centerline.soft = 0
```

Comment:
Since the choices are just `[0, 1]`, I don't think we need an if-else?

```suggestion
param_centerline.soft = arguments.centerline_soft
```

Reply:
You are right! `if-else` removed in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4026/commits/5693e0af70cbe5258e5317ec59df089d87702e30